es/criteria/style.md

```diff
@@ -20,25 +20,25 @@ order: 14
 
 * Ayudar a los colaboradores a escribir bien o a explicar eficazmente lo que hacen.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"